src/stories/table/Table.tsx

```diff
@@ -151,7 +151,7 @@ export function Table<RowData extends Record<string, any>[]>({
             }}
         >
             <thead {...overrides?.thead}>
-                <tr {...overrides?.trHead}>
+                <tr {...overrides?.trHead} className={styles.tableHeader}>
```

Comment:
need to separately add the `overrides.trHead.className` with `clsx` here to ensure className overrides can be added

Reply:
Changed, but check that I added it correctly?